mne/event.py

```diff
@@ -593,9 +604,18 @@ def find_events(raw, stim_channel=None, output='onset',
         ...                   min_duration=0.002))
         [[ 1  0 32]]
 
-    For the digital mask, it will take the binary representation of the
-    digital mask, e.g. 5 -> '00000101', and will block the values
-    where mask is one, e.g.::
+    For the digital mask, if mask_type is set to 'and' it will take the
+    binary representation of the digital mask, e.g. 5 -> '00000101', and will
+    allow the values to pass where mask is one, e.g.::
+
+              7 '0000111' <- trigger value
+             37 '0100101' <- mask
+         ----------------
+              5 '0000101'
+
+    For the digital mask, if mask_type is set to 'not_and' it will take the
+    binary representation of the digital mask, e.g. 5 -> '00000101', and will
+    block the values where mask is one, e.g.::
```

Comment:
In your case you won't use the decorator, but set the parameter whose value will change to `None`. If it's `None` give a warning that it is currently X but will change to Y